phpstan/constants.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Constants defined in this file are to help phpstan analyze code where constants outside the plugin (WordPress core constants, etc) are being used.
+ *
+ * @package WPGraphQL\ContentBlocks
+ */
+
+define( 'WPGRAPHQL_CONTENT_BLOCKS_DIR', '' );
```

Comment:
I am intentionally only defining this one and not (the seemingly identical) `WPGRAPQL_CONTENT_BLOCKS_PLUGIN_DIR`, so this issue is exposed by PHPStan level 1 and a decision can be made whether to dedupe.

Reply:
@justlevine what does this file give us? It seems to work without it.

Reply:
@mindctrl it should however be ignored by phpcs. Are you still actively working on this or is it safe for me to push a commit?

Reply:
@justlevine go ahead and push!